javascript/1-Intro-to-programming/hw-1/scripts.js

```diff
@@ -1 +1,2 @@
 // Your JS code is here
+var name = "tolgonai"
```

Comment:
test looks good, now u can add the actual code.

Reply:
I pull requested with a new request..